server/tinylicious/package.json

```diff
@@ -33,7 +33,7 @@
     "@fluidframework/common-utils": "^0.32.1",
     "@fluidframework/gitresources": "^0.1033.0-40116",
     "@fluidframework/protocol-base": "^0.1033.0-40116",
-    "@fluidframework/protocol-definitions": "^0.1025.0",
+    "@fluidframework/protocol-definitions": "^0.1026.0-0",
```

Comment:
i think tinylicious has to come after server. you don't actually need to update it. someone else will do it when they want to bump tinylicious

Reply:
Okay. Removed those.